do_droplet_import_test.py

```diff
@@ -0,0 +1,128 @@
+import time
+import unittest
+from selenium import webdriver
+from selenium.common.exceptions import NoSuchElementException
+
+import settings
+from helpers import (login_nodeconductor, get_driver, create_project, delete_project, create_ssh_key, 
+                    delete_ssh_key, choose_organization, create_resource, delete_resource, create_provider, 
+                    import_resource, unlink_resource, delete_provider)
+
+
+class NodeconductorTest(unittest.TestCase):
+
+    def setUp(self):
+        self.driver = get_driver(settings.site_url)
+        self.is_project_created = False
+        self.is_provider_created = False
+
+    def test_create_delete_project_key_resource(self):
+        print '%s is going to be loggedin.' % settings.username
+        login_nodeconductor(self.driver, settings.username, settings.password)
+        time.sleep(5)
+        # Identify necessary username on loaded main page
+        username_idt_field = self.driver.find_element_by_class_name('user-name')
+        assert username_idt_field.text == settings.user_full_name, 'Error. Another username.'
+        time.sleep(10)
+        print '%s was loggedin successfully.' % settings.username
+        
+        print 'Organization is going to be chosen.'
+        choose_organization(self.driver, settings.nec_organization)
+        time.sleep(10)
+        print 'Organization was chosen successfully.'
+        
+        print 'Project is going to be created.'
+        create_project(self.driver, settings.project_name)
+        time.sleep(10)
+        self.is_project_created = True
+        print 'Project was created successfully'
+        
+        self.is_provider_created = True 
```

Comment:
This line duplicates line #37. Actually we need to mark provider as created only if provider existence check succeed(before line#51 in `else` block).